src/sonic-yang-models/tests/files/sample_config_db.json

```diff
@@ -304,7 +304,7 @@
                 "switch_id": "2",
                 "switch_type": "voq",
                 "max_cores": "8",
-                "sub_role": "FrondEnd",
```

Comment:
[10:18 PM] Praveen Chaudhary
in this file: src/sonic-yang-models/tests/files/sample_config_db.json

[10:18 PM] Praveen Chaudhary
we need global {} config for test to exercise this code 

[10:18 PM] Praveen Chaudhary
plz add that and we are good